### ISSUE TITLE
Could we support One Plus please?

### ISSUE THREAD

**Issue by sunil-chandran** [leolin310148/ShortcutBadger#95]
Now that One Plus Model 3 has been rated as the [#2 Android phone](http://www.msn.com/en-in/money/gadgets/the-best-android-smartphones-of-2016/ar-AAi6cph#page=4), could you please add support for it?
Best,
S

**Comment by frankboh** [comment]
Yes, please support the OnePlus 3! That would be very liked by the community!

**Comment by Farbklex** [comment]
The question is: Does the default launcher on the One Plus 3 support the badges? If not, there is nothing the developer can do about it.
